thoth/messaging/cve_provided.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+# thoth-messaging
+# Copyright(C) 2019, 2020 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+
+"""This is Thoth Messaging module for CVEProvidedMessage."""
+
+from typing import Optional
+
+from .message_base import MessageBase, BaseMessageContents
+
+
+class CVEProvidedMessage(MessageBase):
```

Comment:
This message should have new implementation I think, @KPostOffice WDYT?

Reply:
https://github.com/thoth-station/messaging/pull/300 this need to be merged before

Reply:
WDYT @KPostOffice  ?

Reply:
@bissenbay Take a look at the messages in `messaging` now, the overall idea is still similar but requires a few changes

Reply:
thanks, https://github.com/thoth-station/messaging/pull/306/commits/081bab6a92c28d9867ca753c155522ae8de507f1